packages/api/test/services/create_content_display_report.test.ts

```diff
@@ -0,0 +1,63 @@
+import 'mocha'
+import chai, { expect } from 'chai'
+import 'chai/register-should'
+import {
+  createTestPage,
+  createTestUser,
+  createUserWithoutProfile,
+  deleteTestUser,
+  getProfile,
+} from '../db'
+import { createGroup } from '../../src/services/groups'
+import {
+  getUserFollowers,
+  getUserFollowing,
+} from '../../src/services/followers'
+import { StatusType } from '../../src/datalayer/user/model'
+import sinonChai from 'sinon-chai'
+import sinon from 'sinon'
+import * as util from '../../src/utils/sendEmail'
+import { MailDataRequired } from '@sendgrid/helpers/classes/mail'
+import { User } from '../../src/entity/user'
+import { getRepository } from '../../src/entity/utils'
+import { ContentDisplayReport } from '../../src/entity/reports/content_display_report'
+import { saveContentDisplayReport } from '../../src/services/reports'
+import { ReportType } from '../../src/generated/graphql'
+import { createTestElasticPage } from '../util'
+import { Page } from '../../src/elastic/types'
+
+chai.use(sinonChai)
+
+describe('saveContentDisplayReport', () => {
+  let user: User
+  let page: Page
+
+  before(async () => {
+    user = await createTestUser('fakeContentUser')
+    page = await createTestElasticPage(user.id)
+  })
+
+  after(async () => {
+    await getRepository(ContentDisplayReport).delete({
```

Comment:
We need to use the superuser here to delete `ContentDisplayReport` because it has the Row level security enabled

Reply:
I think if we delete `user`, it will also delete the `ContentDisplayReport` since it's `onDeleteCascade`